### ISSUE TITLE
A few cats don't have any tags. 

### ISSUE THREAD

**Issue by dinosw** [vtvz/bc-godfat-qol#1]
I noticed that the following cats don't have a tag:

- Miter Saw Cat
- Delinquent Cat
- Cutter Cat
![Screenshot_20241111-175441_Accessibility](https://github.com/user-attachments/assets/622c6440-f139-4fb5-a038-217586414f09)
![Screenshot_20241111-175511_Firefox](https://github.com/user-attachments/assets/05fc8f93-7de8-45e1-951f-b1bdb4fb283d)
![Screenshot_20241111-175523_Firefox](https://github.com/user-attachments/assets/dd9e3638-d41f-409c-b66b-6a5526a1694d)

**Comment by vtvz** [comment]
that's right. They aren't listed on [this chart](https://imgur.com/a/np-charts-9rAfl93). I use it as a reference, which isn't complete. I'm not an expert, and if you have an opinion on what to do with them, let me know

btw, thanks for the first issue. I really appreciate you using it :smiling_face_with_tear:
